src/imu/imu.cpp

```diff
@@ -45,34 +50,53 @@ bool Imu::setup(yarp::os::Property& property) {
         
     yarp::os::Property controlBoardOptions;
     controlBoardOptions.put("device", "remotecontrolboardremapper");
+
+    yarp::os::Bottle remoteControlBoards;
+    yarp::os::Bottle & remoteControlBoardsList = remoteControlBoards.addList(); 
+    yarp::os::Bottle *inputControlBoards;
+    inputControlBoards = property.find("controlboards").asList();
+    for (int i = 0; i < inputControlBoards->size(); i++)
+    {
+        remoteControlBoardsList.addString("/"+robotName+"/"+inputControlBoards->get(i).asString());
+        std::cout << inputControlBoards->get(i).asString() << std::endl;
+    }
+
     yarp::os::Bottle axesNames;
     yarp::os::Bottle & axesList = axesNames.addList();
-    axesList.addString("neck_pitch");
-    axesList.addString("neck_roll");
-    axesList.addString("neck_yaw");
-    axesList.addString("torso_pitch");
-    axesList.addString("torso_roll");
-    axesList.addString("torso_yaw");
-    axesList.addString("l_shoulder_pitch");
-    axesList.addString("l_shoulder_roll");
-    axesList.addString("l_shoulder_yaw");
-    axesList.addString("l_elbow");
-    axesList.addString("r_shoulder_pitch");
-    axesList.addString("r_shoulder_roll");
-    axesList.addString("r_shoulder_yaw");
-    axesList.addString("r_elbow");
 
-    controlBoardOptions.put("axesNames", axesNames.get(0));
+    yarp::dev::PolyDriver tmpDriver;
+    yarp::dev::IEncoders* tmpEncoders;
+    yarp::dev::IAxisInfo* tmpAxis;
```

Comment:
Since these are used also in the `run` function maybe it makes sens make it member of the class